index.html

```diff
@@ -0,0 +1,535 @@
+        </ul>
+
+        <div class="desktop-normal-button">
+            <button type="button" class="btn2">
+                <img src="./images/NormalButton2/Secondary/Icons/Union.png" class="desktop-disabled-button" alt="Arrow down icon">
+            </button>
+        </div>
+    </section>
+
+    <!-- Desktop version work section -->
+    <section class="desktop-work">
+        <div class="desktop-recent-works">
+            <p class="desktop-primary-text">
+                My Recent Works
+            </p>
+        </div>
+
+        <div class="desktop-card-works">
+            <div class="desktop-card-work-row-1">
+                <div class="desktop-card-1">
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
+                    </div>
+                    </div>
+                </div>
+
+                <div class="desktop-card-1">
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
+                    </div>
+                    </div>
+                </div>
+
+
+                <div class="desktop-card-1">
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
```

Comment:
-  You have done great so far :+1: :+1: But there are some requirements you need to complete before moving to the next milestone. :+1: 
> Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

- [ ] According to the project requirement, you are not meant to create a whole new HTML markup for the desktop screen, you already have the full code for the website when you were building out the mobile version, What you were required to do is to use media query in your CSS to dynamically style the website depending on the screen size. I'll suggest you delete the desktop code `from line 208 to 533` in your index.html, then update the CSS code to style the page depending if the user is in a desktop or mobile screen :+1: :+1:

Reply:
I have written seperate classes for the desktop and mobile version to be used for each respective sections so i cant figure out how it will work if i remove the whole script of the desktop version please suggest. As i followed the same procedure for all my responsive projects in my previous model and the codes got approved i didnt tried a different approach

Reply:
> * You have done great so far 👍 👍 But there are some requirements you need to complete before moving to the next milestone. 👍
> 
> > Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).
> 
> * [ ]  According to the project requirement, you are not meant to create a whole new HTML markup for the desktop screen, you already have the full code for the website when you were building out the mobile version, What you were required to do is to use media query in your CSS to dynamically style the website depending on the screen size. I'll suggest you delete the desktop code `from line 208 to 533` in your index.html, then update the CSS code to style the page depending if the user is in a desktop or mobile screen 👍 👍

- [ ] Just like the previous reviewer mentioned, having a single markup for both mobile and desktop while using CSS media query to make it responsive, follows the DRY principle. Using your method for a multi-paged website will be very aching.

<img width="636" alt="image" src="https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/41163571/185dbb09-da4a-476b-acdf-ce9fab8c787f">